### DISCUSSION TITLE
Permitir card titles sin Hx de accesibilidad

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1815] [Proposals]
### Goal

¿Sería posible habilitar la opción de no tener nivel de Hx en las cards?

### Context

Para el proyecto de accesibilidad de Vivo tenemos un caso en el que una lista de entrypoint se muestra como un mosaico vertical de data cards. En su propuesta de accesibilidad esos entrypoint lists son simplemente links, pero Mistica les mete un Hx en el título.

https://www.figma.com/design/9F3lfqy9P8lnhQ8U9kOaT5/Start-Tab----Novo-App?node-id=5029-739632&t=o9yNvKPuMHG4Rxtl-4

### Impact

Conseguir crear la experiencia de accesibilidad planteada por Vivo

### Discussion Questions

-¿Tiene sentido lo que plantea Vivo?
-¿Puede un carrousel de snap cards tener más sentido como links que como card? Por ejemplo, un carrousel de snap cards que son acciones, ahí quizas tendria sentido no mostrar Hx y que se lean como links

**Comment by aweell** [comment]
Hi @MontseTelefonica! Thank you for submitting the discussion.

This issue will be covered in https://github.com/Telefonica/mistica-design/issues/1493
